Assets/Scripts/Possession/Throwable.cs

```diff
@@ -93,7 +93,11 @@ public void Unpossess()
 
     private void ThrowObject()
     {
-        _rb.AddForce(_aim * _throwForce, ForceMode.Impulse);
+        
+        if(_rb.velocity.magnitude < _maxThrowSpeed)

```

Comment:
This only checks your speed which doesn't account for low speeds while in the air. I suggest adding some form of grounded check.